lib/bootloader_setup.pm

```diff
@@ -383,7 +383,8 @@ sub uefi_bootmenu_params {
     assert_screen("grub2-enter-edit-mode", 30) if is_jeos;
     # Kiwi in TW uses grub2-mkconfig instead of the custom kiwi config
     # Locate gfxpayload parameter and update it
-    if (is_jeos && (is_tumbleweed || is_sle('>=15-sp1') || is_leap('>=15.1'))) {
+    if (is_jeos && ((!is_sle('<15-sp1') && !is_leap('<15.1')) || check_var('DISTRI', 'opensuse'))) {
```

Comment:
The oldest supported jeos based on sle is 12-sp5, hence it can be `is_jeos && !is_sle('=12-SP5') || is_opensuse`